src/platform_impl/ios/window.rs

```diff
@@ -196,7 +196,10 @@ impl Inner {
                     let () = msg_send![uiscreen, setCurrentMode: video_mode.video_mode.screen_mode];
                     uiscreen
                 }
-                Some(Fullscreen::Borderless(monitor)) => monitor.ui_screen() as id,
+                Some(Fullscreen::Borderless(Some(monitor))) => monitor.ui_screen() as id,
+                Some(Fullscreen::Borderless(None)) => {
+                    self.current_monitor_inner().ui_screen() as id
+                }
```

Comment:
Could in theory combine these and do something like `monitor.unwrap_or_else(|_| self.current_monitor_inner()).ui_screen() as id` maybe?

Reply:
Yeah, it should work.